configs/detr/README.md

```diff
@@ -24,4 +24,4 @@ We provide the config files for DETR: [End-to-End Object Detection with Transfor
 
 | Backbone | Model | Lr schd | Mem (GB) | Inf time (fps) | box AP | Config | Download |
 |:------:|:--------:|:-------:|:--------:|:--------------:|:------:|:------:|:--------:|
-| R-50 | DETR  |150e |7.9|  | 40.1 | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/detr/detr_r50_8x2_150e_coco.py) | [model](http://download.openmmlab.com/mmdetection/v2.0/detr/detr_r50_8x2_150e_coco/detr_r50_8x2_150e_coco_20201130_194835-2c4b8974.pth) &#124; [log](http://download.openmmlab.com/mmdetection/v2.0/detr/detr_r50_8x2_150e_coco/detr_r50_8x2_150e_coco_20201130_194835.log.json) |
+| R-50 | DETR  |150e |7.9|  | 40.1 | [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/detr/detr_r50_8x2_150e_coco.py) | [model](http://download.openmmlab.com/mmdetection/v2.0/detr/detr_r50_8x2_150e_coco/detr_r50_8x2_150e_coco_20210417_195652-ab96448c.pth) &#124; [log](http://download.openmmlab.com/mmdetection/v2.0/detr/detr_r50_8x2_150e_coco/detr_r50_8x2_150e_coco_20210417_195652-ab96448c.log.json) |
```

Comment:
Add note to indicate that the best results may not be achieved in the last epoch